src/helpers.js

```diff
@@ -22,15 +25,10 @@ export async function normalizeConfig(givenConfig: ConfigGiven): Promise<Config>
     if (typeof privateKey !== 'string') {
       throw new Error('config.privateKey must be a string')
     }
-    if (!(privateKey.includes('BEGIN') && privateKey.includes('KEY'))) {
-      try {
+    if (FS.existsSync(privateKey)) {
         config.privateKey = await readFile(privateKey, 'utf8')
-      } catch (error) {
-        if (error.code === 'ENOENT') {
-          throw new Error(`config.privateKey does not exist at ${privateKey}`)
-        }
-        throw error
-      }
+    } else {
+      config.privateKey = await readString(privateKey)
```

Comment:
Instead of passing it through a no-op function declared at https://github.com/steelbrain/node-ssh/pull/42/files#diff-b38f7254be1a9f817fd2e33c294481a1R10
We can just do `config.privateKey = privateKey`